specification/paths/Suggest-hscode.json

```diff
@@ -0,0 +1,104 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "landed_cost"
+        ]
+      }
+    ],
+    "summary": "Get hs code suggestion.",
+    "description": "This endpoint returns the hs code suggestions for the supplied product.",
+    "requestBody": {
+      "description": "The required fields for the suggestion",
+      "required": true,
+      "content": {
+        "application/json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "properties": {
+              "data": {
+                "required": [
+                  "clientSKU",
+                  "clientDescription"
```

Comment:
I saw this in the landed cost PR but I thought it was for fields we send to hurricane directly. I don't think it's good that we use snake case in the entire spec and camel case here. Also the "client" prefix is somewhat  unnecessary. Same goes for "source". We should really stick to field names that are used in other endpoints too (recipient instead of client?). 

I assume that this is also a problem affecting the frontend code and not only the spec and landed cost. Lets fix it now so it is not too late.